Detectors/ITSMFT/MFT/calibration/src/NoiseCalibratorSpec.cxx

```diff
@@ -0,0 +1,181 @@
+
+    if (mCalibrator->processTimeFrame(digits, rofs)) {
+      LOG(INFO) << "Minimum number of noise counts has been reached !";
+      sendOutput(pc.outputs());
+      pc.services().get<ControlService>().readyToQuit(QuitRequest::All);
+    }
+  } else {
+    const auto compClusters = pc.inputs().get<gsl::span<o2::itsmft::CompClusterExt>>("compClusters");
+    gsl::span<const unsigned char> patterns = pc.inputs().get<gsl::span<unsigned char>>("patterns");
+    const auto rofs = pc.inputs().get<gsl::span<o2::itsmft::ROFRecord>>("ROframes");
+
+    if (mCalibrator->processTimeFrame(compClusters, patterns, rofs)) {
+      LOG(INFO) << "Minimum number of noise counts has been reached !";
+      sendOutput(pc.outputs());
+      pc.services().get<ControlService>().readyToQuit(QuitRequest::All);
+    }
+  }
+}
+
+void NoiseCalibratorSpec::sendOutput(DataAllocator& output)
+{
+  mCalibrator->finalize();
+
+  long tstart = mStart;
+  if (tstart == -1) {
+    tstart = o2::ccdb::getCurrentTimestamp();
+  }
+  long tend = mEnd;
+  if (tend == -1) {
+    constexpr long SECONDSPERYEAR = 365 * 24 * 60 * 60;
+    tend = o2::ccdb::getFutureTimestamp(SECONDSPERYEAR);
+  }
+
+  auto toKeyValPairs = [](std::vector<std::string> const& tokens) {
+    std::vector<std::pair<std::string, std::string>> pairs;
+    Str strutils;
+    for (auto& token : tokens) {
+      //      auto keyval = splitString(token, '=');
+      auto keyval = Str::tokenize(token, '=', false);
+      if (keyval.size() != 2) {
+        // LOG(FATAL) << "Illegal command-line key/value string: " << token;
+        continue;
+      }
+
+      strutils.trim(keyval[1]);
+      std::pair<std::string, std::string> pair = std::make_pair(keyval[0], keyval[1]);
+      pairs.push_back(pair);
+    }
+
+    return pairs;
+  };
+  std::map<std::string, std::string> meta;
+  //  auto keyvalues = toKeyValPairs(splitString(mMeta, ';', true));
+  auto keyvalues = toKeyValPairs(Str::tokenize(mMeta, ';', true));
+
+  // fill meta map
+  for (auto& p : keyvalues) {
+    meta[p.first] = p.second;
+  }
+
+  long startTF, endTF;
+
+  const auto& payload = mCalibrator->getNoiseMap();
+
+  o2::ccdb::CcdbObjectInfo info(mPath, "NoiseMap", "noise.root", meta, tstart, tend);
+  auto flName = o2::ccdb::CcdbApi::generateFileName("noise");
+  auto image = o2::ccdb::CcdbApi::createObjectImage(&payload, &info);
+  info.setFileName(flName);
+  LOG(INFO) << "Sending object " << info.getPath() << "/" << info.getFileName()
+            << " of size " << image->size()
+            << " bytes, valid for " << info.getStartValidityTimestamp()
+            << " : " << info.getEndValidityTimestamp();
+
+  using clbUtils = o2::calibration::Utils;
+  output.snapshot(
+    Output{clbUtils::gDataOriginCLB, clbUtils::gDataDescriptionCLBPayload, 0}, *image.get());
+  output.snapshot(
+    Output{clbUtils::gDataOriginCLB, clbUtils::gDataDescriptionCLBInfo, 0}, info);
```

Comment:
Please note that the convention for the messages sent to CCDB populator was changed since https://github.com/AliceO2Group/AliceO2/pull/6150 in order to avoid clashes between the inputs from different calibrators. The documentation was not fully updated, just pushed https://github.com/AliceO2Group/AliceO2/pull/6234.
Now it should be  e.g.
````
output.snapshot(Output{clbUtils::gDataOriginCDBPayload, "MFT_NoiseMap", 0}, *image.get());
output.snapshot(Output{clbUtils::gDataOriginCDBWrapper, "MFT_NoiseMap", 0}, info);
````

Reply:
Hi Ruben, thanks for the comments. This is now corrected.